my-app/src/app/pages/my-profile/my-profile.component.html

```diff
@@ -51,30 +51,28 @@ <h2>My Profile</h2>
       </div>
     </div>
     <div class="xpay-accounts">
-      <div class="xPay-accounts-edit">
-        <h2>My Pay accounts</h2>
-      </div>
-
-      <div class="account">
-        <h3>Payment accounts</h3>
-        <ul *ngIf="paymentMethods && paymentMethods.length > 0">
-          <li *ngFor="let method of paymentMethods">
-            <p>{{ method.type }}</p>
-            <p>{{ method.card_number }}</p>
-          </li>
-        </ul>
-        <p *ngIf="!paymentMethods || paymentMethods.length === 0">No payment accounts available</p>
-      </div>
-
+      <h2>My xPay accounts</h2>
+      <ul>
+        <li *ngFor="let method of paymentMethods">
+          <div class="payment-method-block">
+            <div  class="payment-method-text">
+              <span>{{ method.type }}</span>
+              <span>{{ method.card_number }}</span>
+            </div>
+            <button class="soon-btn">{{ method.isActive ? 'Block' : 'Unlock' }}</button>
+          </div>
+        </li>
+      </ul>
     </div>
+
     <div class="reservations">
       <div class="reservations-filter">
         <h2>My reservations</h2>
       </div>
 
       <div *ngFor="let ticket of tickets" class="reservation">
```

Comment:
rename ticket to reservation

Reply:
all variables names will be renamed later when API will be ready to implement in on this page. For now we use just mock data to show reservations on my profile page so this component just in development stage.I will do refactor after implemented API